orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/TemplatedPipelineModelMutator.java

```diff
@@ -165,7 +164,7 @@ private void applyConfigurations(
           TemplateMerge.mergeDistinct(
               pipelineTemplateObjectMapper.convertValue(
                   pipeline.get("expectedArtifacts"),
-                  new TypeReference<List<ExpectedArtifact>>() {}),
+                  new TypeReference<List<HashMap<String, Object>>>() {}),
```

Comment:
what was the reason for removing the ExpectedArtifact here?

Reply:
I've committed this change in its own commit within the PR so it'd be documented: https://github.com/spinnaker/orca/pull/3333/commits/27ed129ce3c53f72de3131580e49fda54426fe42
The test added in the commit would fail without this code change.

Reply:
Thanks!